### ISSUE TITLE
Training costs of DiffIR

### ISSUE THREAD

**Issue by Luciennnnnnn** [Zj-BinXia/DiffIR#9]
Hi, great work!

I‘m wondering what's the training costs (days, GPUs, GPU type that used) of DiffIR in evaluated tasks (inpainting, super-resolution, debluring)?

**Comment by Zj-BinXia** [comment]
I don't remember exactly. About 8V100 6-10 days

**Comment by Luciennnnnnn** [comment]
@Zj-BinXia Ok, thank you for your intermediate response.

**Comment by Luciennnnnnn** [comment]
@Zj-BinXia What is the GPU memory of your V100? 32G?

**Comment by Zj-BinXia** [comment]
Yes

**Comment by tzayuan** [comment]
Hi, @Zj-BinXia 
    
1. I am also reproducing the deblurring model S1 stage, but using the default configuration shown below will result in some mini_ batch_ insufficient graphics memory in size(such as batch_size 8 and gt_size 192), my configuration is the same as yours, 8 * V100 32GB. May I ask what may be causing this?

```
    batch_size_per_gpu: 8
    mini_batch_sizes: [8,4,2,2,1,1]             # Batch size per gpu
    iters: [92000,80000,38000,33000,33000,24000]
    gt_size: 512   # Max patch size for progressive training
    gt_sizes: [192, 256,320,384,464,512]  # Patch sizes for progressive training.
```

2. Afterwards, I modified the batch_ The resolution of size and patch, such as batch in the first stage_ Set size to 5 (40 for 8 cards), resolution to 192, and the model can train normally. However, the model will suddenly experience loss and PSNR crashes in one of the tens of thousands of iters. I tried to restore to the previous normal checkpoint, but the problem was temporarily resolved. However, the subsequent iters encountered the same problem again. Have you encountered the same problem during the training process? How did you solve it? (such as dynamically manually adjusting learning rates, backtracking training parameters, etc.)
